combat-trainer.lic

```diff
@@ -680,6 +710,9 @@ class SpellProcess
     @necromancer_healing = settings.necromancer_healing
     echo("  @necromancer_healing: #{@necromancer_healing}") if $debug_mode_ct
 
+    @necromancer_zombie = settings.waggle_sets['zombie']
```

Comment:
I'm not sure how I feel about having `combat-trainer` own this buff set; vs having `buff` execute it. I guess the latter would be pretty complicated...

Reply:
It might be.. `buff` would have to know how to perform the rituals

Reply:
I'm imagining the solution as a set of common necromancy methods, though combat-trainer is still needed for making the corpse